$MirOS: ports/misc/mc/patches/patch-src_treestore_h,v 1.1.7.1 2005/03/18 15:50:29 tg Exp $
--- src/treestore.h.orig	Sun Aug 29 16:57:04 2004
+++ src/treestore.h	Mon Feb 20 18:32:47 2006
@@ -3,8 +3,8 @@
 
 /* Default filenames for the tree */
 
-#define MC_TREE ".mc/Tree"
-#define MC_TREE_TMP ".mc/Tree.tmp"
+#define MC_TREE ".etc/mc/Tree"
+#define MC_TREE_TMP ".etc/mc/Tree.tmp"
 
 typedef struct tree_entry {
     char *name;			/* The full path of directory */
